doc/specs/#653 - Quake Mode/#653 - Quake Mode.md

```diff
@@ -0,0 +1,629 @@
+  - This is `[ { "keys": "ctrl+1", monitor": 1 }, { "keys": "ctrl+2", monitor": 2 } ]`
+
+As some additional examples:
+
+```json
+// Go to the MRU window, wherever it is
+{ "keys": "win+1", "monitor":"any", "desktop": null },
+// Since "any" & null are the default values, just placing a single entry here
+// will bind the same behavior:
+{ "keys": "win+1" },
+
+// activate the MRU window, and move it to this desktop & this monitor
+{ "keys": "win+2", "monitor":"toCurrent", "desktop": "toCurrent" },
+
+// activate the MRU window on this desktop
+{ "keys": "win+3", "monitor":"any", "desktop": "onCurrent" },
+
+// Activate the MRU window on monitor 2 (from any desktop), and place it on the
+// current desktop. If there isn't one on monitor 2, make a new one.
+{ "keys": "win+4", "monitor": 2, "desktop": "toCurrent" },
+
+// Activate the MRU window on monitor 3 (ONLY THIS desktop), or make a new one.
+{ "keys": "win+5", "monitor": 3, "desktop": "onCurrent" },
+
+// Activate the MRU window on this monitor (from any desktop), and place it on
+// the current desktop. If there isn't one on this monitor, make a new one.
+{ "keys": "win+6", "monitor": "onCurrent", "desktop": "toCurrent" },
+```
+
+#### Other properties
+
+Some users would like the terminal to just appear when the global hotkey is
+pressed. Others would like the true quake-like experience, where the terminal
+window "slides-in" from the top of the monitor. Furthermore, some users would
+like to configure the speed at which that dropdown happens. To support this
+functionality, the `globalSummon` action will support the following property:
+
+* `"dropdownDuration": float`
+  - When omitted, `0`, or a negative number: (_default_) No animation is used
+    when summoning the window. The summoned window is focused immediately where
+    it is.
+  - When a positive number is provided, the terminal will use that value as a
+    duration (in seconds) to slide the terminal into position when activated.
+
+We could have alternatively provided a `"dropdownSpeed"` setting, that provided
+a number of pixels per second. In my opinion, that would be harder for users to
+use correctly. I believe that it's easier for users to mentally picture "I'd
+like the dropdown to last 100ms" vs "My monitor is 1504px tall, so I need to set
+this to 15040 to make the window traverse the entire display in .1s"
+
+Some users might want to be able to use the global hotkey to hide the window
+when the window is already visible. This would let the hotkey act as a sort of
+global toggle for the Terminal window. Others might not like that behavior, and
+just want the action to always bring the Terminal into focus, and do nothing if
+the terminal is already focused. To facilitate both these use cases, we'll add
+the following property:
+
+* `"hideWhenVisible": bool`
+  - When `true`: (_default_) When this hotkey is pressed, and the terminal
+    window is currently active, minimize the window.

```

Comment:
> minimize the window

Can we make it such that the terminal slides back up and off the monitor? Like in iTerm2?

Reply:
I'll add a note